tools/devworkspace-generator/src/main.ts

```diff
@@ -45,12 +48,11 @@ export class Main {
     if (!devfilePath && !devfileUrl && !params.devfileContent) {
       throw new Error('missing devfilePath or devfileUrl or devfileContent');
     }
-    if (editorEntry && !pluginRegistryUrl) {
```

Comment:
Could you explain please why do we need these changes

Reply:
We cannot do ```initBindings``` without ```pluginRegistryUrl``` at all.
 That is why I changed this condition:
 
 ```if (editorEntry && !pluginRegistryUrl) {``` -> ```if (!pluginRegistryUrl) {```